src/main/java/randoop/util/Report.java

```diff
@@ -112,18 +114,18 @@ private void generateCSV(){
             PrintWriter writer = new PrintWriter(new File("methods_report.csv"));
 
             StringBuilder sb = new StringBuilder();
-            sb.append("Methods called");
-            sb.append(',');
-            sb.append("Number of times");
-            sb.append('\n');
+            sb.append("Methods called" + ',');
+            sb.append("Number of times" + ',');
+            sb.append("Number of times (Normalized)" + '\n');
 
             for (Map.Entry<Operation, Integer> entry : methodsCalled.entrySet()) {
                 Operation operation = entry.getKey();
                 Integer value = entry.getValue();
```

Comment:
The comment regarding variable rename applies here as well; instead of value, it could be renamed to numberMethodCalls.